snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedReceivedSnapshot.java

```diff
@@ -60,33 +58,30 @@ public long index() {
   }
 
   @Override
-  public ActorFuture<Boolean> apply(final SnapshotChunk snapshotChunk) throws IOException {
-    return actor.call(() -> applyInternal(snapshotChunk));
+  public ActorFuture<Void> apply(final SnapshotChunk snapshotChunk) {
+    return actor.call(
+        () -> {
+          applyInternal(snapshotChunk);
+          return null;
```

Comment:
Why not just used `actor.submit`? I mean, we don't care about a return value, so call is probably not necessary.

Reply:
Submit takes a Runnable which wouldn't allow `applyInternal` to throw an exception. That's also why there's a seemingly useless `return null` there. It's there to cast the lambda to Callable instead of Runnable

Reply:
Ah, I see, you want to be able to throw. We have the concept of a `CheckedRunnable`, but I don't think we use it in the actor scheduler. I think that'd be more than a little bit of refactoring, so let's go with call. :+1: